Assets/_Project/ScriptableObjects/Managers/Editor/EditorTileSelectionManager.asset

```diff
@@ -0,0 +1,19 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
EditorTileSelectionManager::ActiveGroup

![image](https://user-images.githubusercontent.com/1744957/232040566-fe1f3360-40c0-41db-af5d-ce29359a638a.png)

Can this be empty by default and it just selects the first group from the group pool? I dont mind having the option of specifying an active group that is different from the first in the pool, I'm just checking if it is required.

Reply:
Oh this doesnt actually work.. If I select the active group as group1 and load the editor it automatically switches it to group0.

this behaviour is fine, I might just make the field not serialised so it doesnt get shown.